roles/privacyidea_ha/tasks/prepare.yml

```diff
@@ -0,0 +1,51 @@
+---
+# Install packages, define variables and groups, basic ufw configuration and run checks
+
+- name: Install packages
+  apt:
+    name: "{{ privacyidea_packages }}"
+    state: present
+    update_cache: yes
+
+- name: Set unique name of ha cluster (used for grouping)
```

Comment:
Why is this done with a set_fact task instead of setting it in the defaults/host/group vars?

Reply:
Good question, I can't answer, code stolen from squid_proxy_ha